changelog/_unreleased/2025-08-22-fix-customer-registration-address-validation.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Fix incorrect validation handling in customer registration endpoint
+issue: NEXT-11842
```

Comment:
you're not actually referencing a Jira issue here (which would be the `NEXT-` syntax). Just the number for the GitHub issue or with a `#` as a prefix would be more fitting.